range_queries/sparse_table.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+Implementation of Sparse Table
+
+Running Time Complexity
+Build : O(NlogN)
+Range Query : O(1)
+*/
+
+
+#include<vector>
+#include<iostream>
+#include<algorithm>
+
+void computeLogs(int n, std::vector<int> &logs) {   // Util function to store pre-computed logs
+    logs[1] = 0;
+    for (int i = 2 ; i < n ; i++) {
+        logs[i] = logs[i/2] + 1;
+    }
+}
+
+void buildTable(int n, const std::vector<int> &A, const std::vector<int> &logs, std::vector<std::vector<int> > &table) {    // To build Sparse Table data structure
```

Comment:
Line too long.

Reply:
What should I do? Can't use global and I need those variables in the buildTable function.

Reply:
@arpanmukherjee, you can enable `Github Actions` on your fork of this repo. It will take care of the formatting for you.